x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/MlAssignmentPlannerUpgradeIT.java

```diff
@@ -130,12 +130,14 @@ private void waitForDeploymentStarted(String modelId) throws Exception {
 
     @SuppressWarnings("unchecked")
     private void assertOldMemoryFormat(String modelId) throws Exception {
+        // There was a change in the MEMORY_OVERHEAD value in 8.3.0, see #86416
+        long memoryOverheadMb = Version.fromString(UPGRADE_FROM_VERSION).onOrBefore(Version.V_8_3_0) ? 270 : 240;
```

Comment:
The change went into version 8.2.1 - see https://github.com/elastic/elasticsearch/pull/86419

Reply:
Thanks. I updated teh check